modules.d/95nbd/nbdroot.sh

```diff
@@ -112,7 +112,7 @@ if [ "$nbdport" -gt 0 ] 2>/dev/null; then
     if [ -z "$DRACUT_SYSTEMD" ]; then
         nbd-client "$nbdserver" $nbdport /dev/nbd0 $preopts $opts || exit 1
     else
-        systemd-run --no-block --service-type=forking --quiet \
+        systemd-run --no-block --service-type=oneshot --quiet \
```

Comment:
Why do you think `oneshot` is the solution? This should also go into the commit message.

```
$ fgrep 'Type=' /usr/lib/systemd/system/nbd@.service
Type=forking
```

Reply:
We always follow upstream not never whatever is shipped in downstream right? ( https://github.com/NetworkBlockDevice/nbd/blob/master/systemd/nbd%40.service.sh.in ).

Reply:
So, what I did in dracut, was to check the version of the software, so that we can at least support small changes in behavior. Maybe we can switch the option dynamically, if the newer version is detected?

Then fade out support for older versions over time.